core/settings/editor-preferences/model.php

```diff
@@ -238,4 +238,10 @@ protected function register_controls() {
 
 		$this->end_controls_section();
 	}
+
+	private function should_display_checklist_toggle_control() : bool {
+		return Plugin::$instance->experiments->is_feature_active( ChecklistModule::EXPERIMENT_ID ) &&
+			Plugin::$instance->experiments->is_feature_active( AppBarModule::EXPERIMENT_NAME ) &&
+			current_user_can( 'manage_options' );
+	}
```

Comment:
This feels like it belongs as a static public function inside the module perhaps, I could be wrong. WDYT?